src/components/plot/index.tsx

```diff
@@ -229,7 +235,10 @@ export class PlotBase extends React.PureComponent<PlotProps, any> {
   }
 
   private copied() {
-    const copiedIndicator = document.getElementById('copied');
+    const copiedIndicator = document.getElementById(JSON.stringify(this.props.spec));
```

Comment:
Instead of causing side-effect using `document.getElementById`, 
why don't we do things in react way -- e.g., by making the popup depends on a local state that determines if a popup should be shown.